assets/js/googlesitekit/datastore/site/urls.js

```diff
@@ -0,0 +1,134 @@
+/**
+ * `core/site` data store: urls.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { getLocale, getLocaleLanguage } from '../../../util';
+
+export const initialState = {};
+
+export const actions = {};
+
+export const controls = {};
+
+export function reducer( state ) {
+	return state;
+}
+
+export const resolvers = {};
+
+export const selectors = {
+	/**
+	 * Gets an external Google URL that includes the user's locale.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Object} state          Data store's state.
+	 * @param {Object} [args]         Optional arguments for the resulting URL.
+	 * @param {string} [args.website] Base URL hostname and schema.
+	 * @param {string} [args.path]    URL path to build complete URL with starting slash.
+	 * @param {Object} [args.query]   Object to append query to the URL.
+	 * @param {string} [args.hash]    Optional hash.
+	 * @param {string} [args.locale]  Optional locale.
+	 * @return {(string|null)} The URL containing the user's locale or `null` if path is not set.
+	 */
+	getGoogleURL( state, args ) {
```

Comment:
This is a handy selector, and I know it's tested indirectly elsewhere, but can you add a few tests for it? Mainly to serve as a usage reference 🙂

Reply:
Added